src/centralserver/admin-service/application/build.gradle

```diff
@@ -23,6 +23,11 @@ dependencies {
     implementation('org.liquibase:liquibase-core') //TODO enabling runtime liquibase
 
     intTestImplementation("com.nortal.test:test-automation-core:${testAutomationFrameworkVersion}")
+    constraints {
```

Comment:
can we consider adding this to root build.gradle (under src) and using version range instead of contant override? In theory this vulnerability can come from any dependency, no need to concentrate on test automation